Update Constitution.md (#1)
umanitoba-cssa/constitution: Constitution.md

Reviewer: Some general comments about the constitution that is not related to the incoming changes:

In the definitions section of Open Club it says that Open Clubs are open to **ANY** members at U of M. We do not want to be defined as this. It states in section 1c that we are defined as an open club.

Some committees have been defined, are we defining them all them or are we just doing a select few?

Author: defining the main ones we want to have as standard, you also mentioned on leading the outreach committee I just forgot to add that, Having that be specific to student affairs would be ok, or would student affairs have their own standard one?

> The name of the student club shall be Computer Science Students’ Association, also known as CSSA.

Reviewer: ```suggestion
The name of the student club shall be Computer Science Students’ Association, also known as the CSSA.
```

CSSA will consist of the following executives members, who shall be voting members of the group as part of the council:

> - Co-Presidents

Reviewer: Why not President(s)?

> - Director of Finances and Internal Affairs

Reviewer: ```suggestion
- Executive of Finances and Internal Affairs
- Executive of Lounge Affairs
- Executive of Student Affairs
- Executive of Events
- Executive of Advocacy
```

Executive is a defined term with a meaning, while Director doesn't mean anything.

Author: The purpose is the standardized one that has been used for the class presentations along with wics and .devClub

noun
noun: director; plural noun: directors

    a person who is in charge of an activity, department, or organization.
    "he has been appointed finance director"
        a member of the board of people that manages or oversees the affairs of a business.
        
We can use other words, don't have to use director as other synonyms exist. Although some previous motions have already been signed as such, and Executive itself refers to the executive body as a whole but not specific positions. 

Requires more discussion to finalize.

- Director of Lounge Affairs
- Director of Student Affairs
- Director of Events
- Director of Advocacy

CSSA will consist of the following non-executives who shall be non-voting members of the group but will be involved in assisting the executives as part of the council:

### Councillor

1. Non-voting member of council.

> 2. Voting member of a committee led by a Director.

Reviewer: ```suggestion
2. Voting member of a committee led by an Executive.
```

3. Perform any duties directed by their committee.

> Voting Members may not be on academic or social probation.

Reviewer: Lets remove this line

Author: > Lets remove this line

Line 95 about probation?

## Article 2b: Executive Officers and Duties

> ### Co-Presidents

Reviewer: This seems very disingenuous. Was this approved officially during the 2022-2023 council? It kind of felt like this was allowed through a loophole. We also don't want to allow this in the future. I don't also like that we are removing presidents for co-presidents.

The Co-presidents is a transitional role approved by the 2022-2023 council for the 2023-2024 CSSA general elections and elected as a single voting member on council.

1. Act as a signing authority for CSSA.
2. Be the contact person for UMSU.
3. Maintain frequent and punctual communication with the Faculty of Science, Department of Computer Science, UMSU, Members of Council, Members of the Association, and all other relevant parties.
4. Attend Student Executive Assembly (SEA) Meetings on behalf of CSSA.
5. Be responsible for the overall operations of the group.

> 6. Deal with issues concerning the club and its individual Members.

Reviewer: If we are trying to come up with ways to stop make it so that the presidents don't have all of the power then this feels weird. It makes it feel like the president can just penalize execs with executive authority.

7. Serve as signing authority on any group accounts.
8. Be a voting member of CSSA for management decisions.
9. Call special meetings.

> 10. Attend all meetings.

Reviewer: We need to define meetings because I don't think we want this once we have committees.

11. Carry out provisions of the constitution.
12. Ensure the group funds remain in budget with local and corporate sponsors along with the Director of Finances and Internal Affairs.
13. Ensure that the CSSA lounge is a safe and inclusive space.
14. Enforce that the transition report is done by April 30 of the council’s term.
15. Perform any other duties incidental to the office of the President or directed by Council.

> ### Director of Finances and Internal Affairs

Reviewer: ```suggestion
### Executive of Finances and Internal Affairs
```

1. Act as a signing authority for CSSA.
2. Providing a brief report on CSSA finances and expenses at each meeting.
3. Be responsible for all funds generated by Council events and for overseeing all funds spent by Council.
4. Make available all audits of CSSA to any Student members upon request with more than two weeks of notice.
5. Ensure the Council is kept updated on ongoing council initiatives.
6. Perform a year-end audit at the end of the CSSA Fiscal Year.
7. Reserve the meeting room for CSSA meetings, General meetings and the CS Executive Assembly.
8. Prepare an agenda and council package for Regular Meetings and distribute them to the Council at least two days prior to the meeting.
9. Receive resignations of Council and inform Executives and the Computer Science student body of any such resignations.
10. Ensure that the CSSA lounge is a safe and inclusive space.

> 11. Lead the Financial Accounability Committee with up to 3 committee members.

Reviewer: Why do we have an upper limit on the number of committee members. Why not leave it open and the Finance and Internal Affairs exec can decide that?

12. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.

> ### Director of Lounge Affairs

Reviewer: ```suggestion
### Executive of Lounge Affairs
```

1. Act as a signing authority for CSSA.
2. Create a cleaning schedule for the CSSA Lounge.
3. Monitor maintenance of the CSSA Office, CSSA Lounge, and purchase pertinent supplies when needed.
4. Ensure appropriate reservation, organization, and signage for events that take place in the CSSA Lounge.
5. Ensure that the CSSA lounge is a safe and inclusive space.
6. Lead the CSSA lounge supervisors and work with council in appointing them.

> 7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.

Reviewer: Up to Ethan but do we want to add a line about the canteen.

> ### Director of Student Affairs

Reviewer: ```suggestion
### Executive of Student Affairs
```

1. Act as a liaison for the students towards the Department of Computer Science, Faculty of Science and UMSU.
2. Act as a liaison between Students and Council.
3. Stay up to date on departmental and Senate meetings pertaining to the Computer Science program.
4. Lead relevant campaigns on issues affecting the Computer Science Student Community.
5. Work with the Director of Advocacy to promote opportunities of initiative/programs/organizations pertaining to where the Computer Science community is underrepresented.
6. Ensure that the CSSA lounge is a safe and inclusive space.
7. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.

> ### Director of Events

Reviewer: ```suggestion
### Executive of Events
```

1. Be responsible for planning the events for the Computer Science community.

> 2. Coordinate team-building events for Council.

Reviewer: Do we really want to make this official?

3. Coordinate a minimum of one event or academic initiative in each term, as per the discretion of the Council.
4. Ensure there are an adequate number of social and academic events.

> 5. Create posters and advertisements for all CSSA events.

Reviewer: Again this is super specific. Is this up to the events exec or is this for the events committee?

Author: It is something they are accountable for, but up to them to decide how they want to split or delegate their responsibility to others.

8. Perform other duties incidental to this office within reasonable discretion as directed by the President and/or Council.

### All Executives

1. Report to council regarding their activities.
2. Participate in voting on club related activities.
3. Participate and assist with Group activities.
4. Be responsible for any specific tasks delegated to them by general resolution of Council.
5. Declare any conflicts of interest.
6. Declare any personal or financial interest in any expenditure of the group funds.
7. Comply with all regulations set out by the Campus Police of the Dean’s Office, specifically those concerning smoking and drinking on campus.
8. Promote Council cohesion and a respectful atmosphere abiding by the University of Manitoba Respectful Workplace and Learning Environment Policy.
9. Refrain from drug/alcohol use or sexual activity at any CSSA event or in the CSSA lounge. Alcohol consumption is only permitted if the venue of the event distributes alcohol.
10. Not allow any persons who are not Members of Council to use CSSA Office space in the absence of a Member of Council.

> 11. Prepare, in cooperation with other executives of the same position when applicable, an orientation package for the subsequent person to fill each position.

Reviewer: There is a whole section on this, we don't need this here.

12. Attend all regular meetings.
13. Put fair effort into acquiring a proxy, in the event a voting member is unable to attend a Council meeting.

> ## Article 2c: Executive Officer Dismissal Protocol

Reviewer: keep in mind of what we put in the duties in article 2b. Because technically if we don't follow it then they should go into probation.
